### PR TITLE
Dynamic GITHUB_URL & Github Action for Security & Linting

### PR DESCRIPTION
**Pull request by Autumn-Ou**
Use github url in all possible places including in text, note md files still need it manually changed if the URL were to change or a md compiler added.

### CODE REVIEW

**Comment by Autumn-Ou** [comment]
Also added a few more links to features panel in this commit

**Comment by Autumn-Ou** [comment]
Now also includes built in security checks and automated linting on every commit

**Comment by Autumn-Ou** [comment]
image

File: components/screens/landing/SignedOut.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ async function getContributors() {
   if (contributors) return contributors;
 
   const fetchContributors = await fetch(
-    "https://api.github.com/repos/gryphonmachine/machine/contributors?per_page=100"
+    GITHUB_URL+"/contributors?per_page=100"
```
Comment (heybereket) [inline components/screens/landing/SignedOut.tsx:17]: Change this to: `${GITHUB_URL}/contributors?per_page=100`

File: components/screens/landing/SignedOut.tsx [hunk 2]
```diff
@@ -91,7 +91,7 @@ export const SignedOutScreen = () => {
           <h1 className="text-xl text-lightGray">
             We&apos;re 100% open-source on{" "}
             <a
-              href="https://github.com/gryphonmachine/machine"
+              href={`${GITHUB_URL}`}
```
Comment (heybereket) [inline components/screens/landing/SignedOut.tsx:94]: change this to `href={GITHUB_URL}`

**Review by heybereket — CHANGES_REQUESTED** [review]
LGTM, made some comments.

**Comment by Autumn-Ou** [comment]
Done

**Pull request merged**